docs/source/UserGuide/GettingStarted/InstallingClient/index.rst

```diff
@@ -11,7 +11,7 @@ Install script
 
 Download the *dirac-install* script from::
 
-  wget -np -O dirac-install https://github.com/DIRACGrid/DIRAC/raw/integration/Core/scripts/dirac-install.py --no-check-certificate
+  wget -np -O dirac-install https://github.com/DIRACGrid/DIRAC/raw/master/Core/scripts/dirac-install.py --no-check-certificate
```

Comment:
The integration branch is where all the new developments should go. We screwed it in the past but now we should get back to this practice.